src/main/java/co/willsalz/ttyl/entities/Representatives.java

```diff
@@ -0,0 +1,21 @@
+package co.willsalz.ttyl.entities;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.hibernate.validator.constraints.NotEmpty;
+
+import java.util.List;
+
+public class Representatives {
```

Comment:
maybe add a `@JsonIgnoreProperties(ignoreUnknown = true)` if we're feeling really paranoid.